content/02.introduction.md

```diff
@@ -114,11 +114,29 @@ Respiratory failure remains the leading cause of death for COVID-19 patients [@d
 Besides major pulmonary damage, SARS-CoV-2 infection can damage many other organ systems including the heart [@doi:10.1001/jamacardio.2020.0950], kidney [@doi:10.1038/s41585-020-0319-7; @doi:10.1186/s13054-020-02872-z], liver [@doi:10/ggpx6s], and gastrointestinal tract [@doi:10.1053/j.gastro.2020.02.055; @doi:10.1111/1751-2980.12851].
 It can even cause neurological complications [@doi:10.1016/j.bbi.2020.03.031]. 
 As it becomes clearer that SARS-CoV-2 infection can damage multiple organs, the scientific community is pursuing multiple avenues of investigation in order to build a consensus about how the virus affects the human body.
-To quickly associate the clinical outcomes with SARS-CoV-2 infection, researchers are taking advantage of “omics” technologies to profile the expression of coronavirus entry factors across the tissues.
-
-
-
-[Start with RNA-seq description and summarize studies so far including healthy and infected tissues/cells in vivo/in vitro]
+To quickly associate the clinical outcomes with SARS-CoV-2 infection, researchers are taking advantage of “omics” technologies to profile the expression of coronavirus entry factors across the tissues/cells and also to measure the host transcriptional response after virus infection. 
+Although we make an effort to evaluate all relevant studies, we realize that this area is rapidly evolving.
+
+Two studies profiled expression response after SARS-CoV-2 infection in multiple human cell lines (Figure X1) [@doi:10.1016/j.cell.2020.04.026; @doi:10.1101/2020.05.05.079194]. 
```

Comment:
I can help get the figure to work in Manubot.  Is this a figure you made?  Do you have the source files that we can also add to the repository?  That lets other contributors tweak it later if needed.

I recommend making the filename a description of the figure instead of a number because the numbering can change.  Perhaps `cell-lines-moi.jpg` or something like that.  Then, in the text where you want to refer to the figure use
```suggestion
Two studies profiled expression response after SARS-CoV-2 infection in multiple human cell lines (Figure {@fig:cell-lines-moi}) [@doi:10.1016/j.cell.2020.04.026; @doi:10.1101/2020.05.05.079194]. 
```

Where you want the figure to appear, add the text (including the blank line before and after):
```

![Caption for the figure here.](images/cell-lines-moi.jpg){#fig:cell-lines-moi}

```

Reply:
Thanks. Yes, I analysed the data and made the figure. Where should I put the source file(s)? It was edit in Affinity.

Reply:
Thanks @bansalvi.  Putting the source files where you did in the images directory is a good choice.

The figure shows up as intended in the manuscript now:
![image](https://user-images.githubusercontent.com/3532898/85223973-908a5680-b38c-11ea-8c4f-fb8ac03cfc2e.png)

You'll have to view the PDF version from AppVeyor to see it.  It's not in the HTML version because the images subdirectory is not copied in these preview builds.

Reply:
@agitter This is great. Thanks. Can I also refer to part of the figures in the text? For example Figure 2A.

Reply:
Yes, to refer to Figure2A in the text, you can write:
```
Figure {@fig:cell-lines-moi}A
```

The panel letters are not automated, so the command will only replace the figure number.